app/models/event.rb

```diff
@@ -175,6 +187,11 @@ def scheduled_kind
     end
   end
 
+  def get_show_deal_path
+    "#{ENV['FRONTEND_URL']}#{account_deal_path(self.account, self.deal)}"
```

Comment:
Aqui da para utilizar em vez do `account_deal_path` o account_deal_url`

```suggestion
    account_deal_url(self.account, self.deal)
```

Reply:
Mas sera que não é melhor chamar o helper diretamente?